cdap-api/src/main/java/co/cask/cdap/api/schedule/Schedule.java

```diff
@@ -50,7 +60,9 @@ public String getDescription() {
 
   /**
    * @return Cron expression for the schedule.
+   * @deprecated As of version 2.8.0, use {@link TimeSchedule#getCronEntry()} instead
```

Comment:
hmmm but TimeSchedule will be an internal class... so we don't want to recommend people to do that. Perhaps here, because we don't want people to use this at all in the future, just say do not use this any more. This is a funny thing about making the Schedule subclasses internal - people can create them but not inspect them...